Editor/Window/RegisterComputeInput.cs

```diff
@@ -96,7 +84,8 @@ private async void OnRegisterComputeButtonClicked()
                     _stateManager.WebSocketUrl = registerResponse.WebSocketUrl;
                     _computeState = ComputeStatus.Registered;
 
-                    if (!string.IsNullOrWhiteSpace(previousComputeName))
+                    var previousComputeName = _stateManager.ComputeName;                 
+                    if (!string.IsNullOrWhiteSpace(previousComputeName) && previousComputeName != _computeName)
```

Comment:
Adding this check here seems better than at the top level of this method. If for some reason the compute is deleted outside of Unity, _computeState is set as NotRegistered, and it's easier this way to ensure that it does end up getting registered.

Reply:
Another alternative is to disable the button if the computeName is same as the previousComputeName, but that adds more scope of changes.